### ISSUE TITLE
Assignment grading screen not working when Accessibility Toolbar is enabled.

### ISSUE THREAD

**Issue by davefoord** [willianmano/moodle-theme_moove#346]
I am using Moodle 3.10.7+ (Build: 20211015) and Moove 3.10.01

If a teacher has enabled the accessibility toolbar, if they go into an assignment to grade a student's piece of work, when you get to the grading screen (the one with the assignment displayed as a pdf on the left, the grading options on the right) the top of screen gets hidden, which hides the student name, and the ability to select different students etc.

Turning off the accessibility toolbar solves the problem. It would however be great if the accessibility toolbar worked on this screen, or if it cannot work, somehow stop the assignment grading screen from 'sliding up' as it does - so the teacher can leave the toolbar enabled for other purposes.

**Comment by willianmano** [comment]
Sorry, I only support the current major version of Moodle, which is 4.0.

**Comment by willianmano** [comment]
@davefoord as Accessibility is a major for me, I am reopening this issue and I will solve it on Moodle 4.0.

**Comment by rachelfeld** [comment]
I am having this same issue on 4.2. Any timetable on when it might be addressed? The accessibility toolbar is fantastic and I'd like to encourage all my users to enable it.

**Comment by carlicockrell** [comment]
I'm also following this. We recommend Moove as one of the themes more focused on accessibility, and we are less likely to use it until this situation is addressed as people have to turn the toolbar off to return to typical navigation.
